rust/lib/src/rustlib.udl

```diff
@@ -0,0 +1,39 @@
+namespace rustlib {
+  string init_logging();
+  string execute_command(string cmd, string args);
```

Comment:
> This is a good start for UniFFI and a good judgement to approach it.
> 
> IDK if the intention is to keep growing this API and start extracting the string commands of `execute_commands` into their own methods or stick with the local client/server approach that is currently being used.
> 
@zancas IDK either. Sometimes this seems handy, because you can include a new command easily in both repos, but maybe it's a good idea to extract some of these commands on their own methods...
> I think the succinct API that the current approach has is convenient in terms of boilerplate code that is needed to expose changes in the rust library. The downside is that the app relies on flying strings. like `execute_command(cmd: "sync", "")` and this returns some other string
> 
The pattern for errors is not the best, we need to change it in some point... Checking the start of the string looking for `error` is really rudimentary... I'll happy to change this, but I need this change first from zingolib.
> A more robust API would having a `rustlib.sync()` that returns a `ReturnType` result instead of a string. The downside is that it envolves coding the intermediate code. Maybe the scaffolding created from the udl file can take care of most of it.
> 
Agreed 100%. I added types of all of this returns from the `command` but only when you parse the string to a json object.
> This can be also remediated by creating a better TypeScript API for interacting with `execute_command` and keep this succinct UniFFI API as is to keep a better velocity and less boiler plate for exposing new functionality on the app.
> 
> The main problem is that the Strings resulting on the responses could be broken from a change in the zingolib without that causing a problem in the build itself.
>
In fact, if you review the code base entirely, you can see sometimes, exceptions about this behavior that make you feel even worse...

Reply:
We need to work in all of these thing, but I don't thing any of them are a blocker. Thanks @pacu